### PR TITLE
Add CreateOrder command

### PR DESCRIPTION
**Pull request by danizord**

### CODE REVIEW

File: src/ServiceDescription/Shopify-v1.php [hunk 1]
```diff
@@ -1899,6 +1899,226 @@
...
+            'data'             => ['root_key' => 'order'],
+            'parameters'       => [
+                'billing_address' => [
+                    'description' => 'The mailing address associated with the payment method',
+                    'location'    => 'json',
+                    'type'        => 'object',
+                    'required'    => false,
+                ],
+                'browser_ip' => [
+                    'description' => 'The IP address of the browser used by the customer when placing the order',
```
Comment (bakura10) [inline src/ServiceDescription/Shopify-v1.php]: Are you sure this one is writable?

File: src/ServiceDescription/Shopify-v1.php [hunk 2]
```diff
@@ -1899,6 +1899,226 @@
...
+                    'type'        => 'string',
+                    'required'    => false,
+                ],
+                'buyer_accepts_marketing' => [
+                    'description' => 'Indicates whether or not the person who placed the order would like to receive email updates from the shop',
+                    'location'    => 'json',
+                    'type'        => 'boolean',
+                    'required'    => false,
+                ],
+                'cancel_reason' => [
```
Comment (bakura10) [inline src/ServiceDescription/Shopify-v1.php]: Cancelling when creating the order?

File: src/ServiceDescription/Shopify-v1.php [hunk 3]
```diff
@@ -1899,6 +1899,226 @@
...
+                    'description' => 'The reason why the order was cancelled',
+                    'location'    => 'json',
+                    'type'        => 'string',
+                    'required'    => false,
+                ],
+                'cancelled_at' => [
```
Comment (bakura10) [inline src/ServiceDescription/Shopify-v1.php]: Are you sure for this one?

File: src/ServiceDescription/Shopify-v1.php [hunk 4]
```diff
@@ -1899,6 +1899,226 @@
...
+                    'description' => 'The date and time when the order was cancelled. If the order was cancelled',
+                    'location'    => 'json',
+                    'type'        => 'string',
+                    'required'    => false,
+                ],
+                'cart_token' => [
```
Comment (bakura10) [inline src/ServiceDescription/Shopify-v1.php]: Are you sure this one is writable?

File: src/ServiceDescription/Shopify-v1.php [hunk 5]
```diff
@@ -1899,6 +1899,226 @@
...
+                    'description' => 'Unique identifier for a particular cart that is attached to a particular order',
+                    'location'    => 'json',
+                    'type'        => 'string',
+                    'required'    => false,
+                ],
+                'client_details' => [
```
Comment (bakura10) [inline src/ServiceDescription/Shopify-v1.php]: Are you sure this one is writable?

File: src/ServiceDescription/Shopify-v1.php [hunk 6]
```diff
@@ -1899,6 +1899,226 @@
...
+                    'description' => 'An object containing information about the client',
+                    'location'    => 'json',
+                    'type'        => 'object',
+                    'required'    => false,
+                ],
+                'closed_at' => [
```
Comment (bakura10) [inline src/ServiceDescription/Shopify-v1.php]: Are you sure this one is writable?

File: src/ServiceDescription/Shopify-v1.php [hunk 7]
```diff
@@ -1899,6 +1899,226 @@
...
+                    'type'        => 'string',
+                    'required'    => false,
+                ],
+                'tags' => [
+                    'description' => 'Tags are additional short descriptors',
+                    'location'    => 'json',
+                    'type'        => 'string',
+                    'required'    => false,
+                ],
+                'landing_site' => [
```
Comment (bakura10) [inline src/ServiceDescription/Shopify-v1.php]: Are you sure this one is writable? I think Shopify uses the referrer ID to populate it

File: src/ServiceDescription/Shopify-v1.php (hunk 6 above)
Comment (danizord) [inline src/ServiceDescription/Shopify-v1.php]: Yep :)

**Pull request merged**